source/features/recently-pushed-branches-enhancements.tsx

```diff
@@ -7,7 +7,7 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 import {getRepo} from '../github-helpers';
 
-const fragmentURL = `/${getRepo()!.nameWithOwner}/show_partial?partial=tree%2Frecently_touched_branches_list`;
+const fragmentURL = `/${getRepo()?.nameWithOwner}/show_partial?partial=tree%2Frecently_touched_branches_list`;
```

Comment:
This should hide the error for the time being. I had thought about functions as well but they looked like a bad patch. Let's merge the 2 `init`s in the next lint/PR